.travis.yml

```diff
@@ -8,13 +8,19 @@ python:
   - 3.6
   - nightly
   - pypy
+cache:
```

Comment:
Just `cache: pip` would work. And also there's this syntax:
```yaml
cache:
  pip: true
```

I don't really like the idea of hardcoding paths.